test/test_core.py

```diff
@@ -5916,25 +5916,24 @@ def test_sigaction_default(self, signal, exit_code, assert_identical):
 
   @no_windows('https://github.com/emscripten-core/emscripten/issues/8882')
   @requires_node
-  def test_unistd_access(self):
+  @parameterized({
+    '': (['-DMEMFS'],),
+    'nodefs': (['-DNODEFS', '-lnodefs.js'],),
+    'noderawfs': (['-DNODERAWFS', '-sNODERAWFS'],)
```

Comment:
Do we need `-l` with nodefs but not noderawfs?

Reply:
They both work.  I'll make them consistent.

Reply:
Actually `-sNODERAWFS` on its own doesn't seem to work right now.    So leaving as is for now.